arcade/gui/property.py

```diff
@@ -18,21 +18,39 @@ class _Obs(Generic[P]):
     def __init__(self, value: P):
         self.value = value
         # This will keep any added listener even if it is not referenced anymore and would be garbage collected
-        self.listeners: Set[Callable[[], Any]] = set()
+        self.listeners: Set[Callable[[Any, Any], Any]] = set()
 
 
 class Property(Generic[P]):
     """
     An observable property which triggers observers when changed.
 
+        def log_change(instance, value):
```

Comment:
Both Paul Craven and I prefer the explicit `.. code-block:: python` style.
```suggestion
... code-block:: python

        def log_change(instance, value):
```